packages/venia-concept/src/webpackUtils.js

```diff
@@ -0,0 +1,39 @@
+const fetch = require('node-fetch');
+
+const fetchQuery = query =>
+    fetch(new URL('graphql', process.env.MAGENTO_BACKEND_URL).toString(), {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({
+            query
+        })
+    })
+        .then(result => result.json())
+        .catch(err => {
+            console.error(err);
+            return '';
+        });
+
+/**
+ * An Async function that will asynchronously fetch the
+ * media backend Url from magento graphql server. Also
+ * will set the MAGENTO_MEDIA_BACKEND_URL value on global
+ * object to the resulting url.
+ *
+ * @returns Promise that will resolve to the media backend url.
+ */
+const getMediaURL = () =>
```

Comment:
I like this better as a utility function than a webpack plugin for sure. Seems also like something that could / should live in `buildpack` instead of `venia-concept` though.